docs/android/v2/features/HLS.mdx

```diff
@@ -11,9 +11,104 @@ Behind the scenes, this will be achieved by having a bot join your room and stre
 > Note that the media server serving the content in this case is owned by 100ms. If you're looking for a way to stream
 on YouTube, Twitch etc., please have a look at our RTMP streaming docs [here](./RTMP-recording).
 
-## Starting HLS
+## Starting 
 
-To start HLS you'll need to pass in a meeting URL. The 100ms bot will open this URL to join your room, so
+HLS can be started in three ways depending on the level of customization you need.
+1. [Default View](#default-view): The simplest view to just begin a stream with default UI and parameters.
+2. Portrait View: The default view streams a landscape video. This can be changed to portrait instead.
+3. Custom Views: To use your own UI for HLS, you would need a custom implementation of the browser that joins to stream it. Provide your own URLs for it to join.
+
+#### Default View
+
+<Tabs id="hls-default-view" items={['Kotlin', 'Java']} />
+
+<Tab id='hls-default-view-0'>
+
+```kotlin
+hmsSdkHost.startHLSStreaming(hmsActionResultListener = object : HMSActionResultListener {
+    override fun onSuccess() {
+        // hls was started successfully
+    }
+
+    override fun onError(error: HMSException) {
+        // hls failed to start with error $error
+    }
+
+})
+```
+
+</Tab>
+
+<Tab id='hls-default-view-1'>
+
+```java
+hmsSdkHost.startHLSStreaming(null, new HMSActionResultListener() {
+    @Override
+    public void onSuccess() {
+        // hls was started successfully
+    }
+
+    @Override
+    public void onError(@NonNull HMSException error) {
+        // hls failed to start with error $error
+    }
+});
+
+```
+
+</Tab>
+
+#### Portrait View
```

Comment:
@AniketSK @PratimMallick my understanding was that the portrait capture automatically happens when the device orientation changes to portrait, and the `metadata` field is actually not used at all. can we confirm this? 

if my understanding is correct, can we remove the portrait view section from this doc and ship it with the "default view" change?

Reply:
@arjunattam This feature is not available currently.